napari/_qt/layer_controls/qt_layer_controls_base.py

```diff
@@ -6,6 +6,9 @@
 from ...utils.translations import trans
 from ..widgets._slider_compat import QDoubleSlider
 
+# opaque and minimum blending do not support changing alpha (opacity)
+NO_OPACITY_BLENDING_MODES = {'minimum', 'opaque'}
```

Comment:
```suggestion
NO_OPACITY_BLENDING_MODES = {str(Blending.MINIMUM), str(Blending.OPAQUE)}
```

This is a small change that changes from using "magic constraints" to enforcing better code consistency (If someone starts to refactor code, it will faster inform a coding person which place should also be visited).  

This change requires also adding `Blending` import from `...layers.base._base_constants` (line 4)

Reply:
I don't think we need the `str()` here.